frontend/src/store/FamLoginUserState.ts

```diff
@@ -106,6 +113,20 @@ const getApplicationsUserAdministers = () => {
     return applicationList;
 };
 
+const isApplicationAdmin = () => {
+    const appsUserIsAdmin = getUserAccess().filter(
+        (access) => access.auth_key == APP_ADMIN_ROLE
+    );
+
+    if (appsUserIsAdmin.length > 0) {
+        appsUserIsAdmin[0].grants.filter(
+            (grant) => grant.application.id == FAM_APPLICATION_ID
+        );
+        return true;
```

Comment:
Hi Jota, we only return true when your filter here has length > 0

so `getUserAccess().filter((access) => access.auth_key == APP_ADMIN_ROLE);` will return you the information like below, and you have to go through the grants, and check each grant, if there is any `application.id== FAM_APPLICATION_ID`. So in order to return true, your filter result needs to have length > 0. 
<img width="735" alt="Screen Shot 2024-02-27 at 12 41 44 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/61b55350-7d9b-4271-964e-45bd48317b48">

```
const userAdminAccess = getUserAccess().find((access) => access.auth_key == APP_ADMIN_ROLE);
const appsUserIsAdmin = userAdminAccess.grants.filter(grant => grant.application.id == FAM_APPLICATION_ID);
if (appsUserIsAdmin.length > 0) return true;
return false;
```

Reply:
just made the change